packages/calcite-components/src/components/avatar/utils.ts

```diff
@@ -21,6 +24,23 @@ export function stringToHex(str: string): string {
   return hex;
 }
 
+/**
+ * The function splits the string into two halves, reverses each half, and then concatenates them.
+ *
+ * @param {string} string - The input string to be mixed.
+ * @returns {string} - The mixed string.
+ */
+function mixStringDeterministically(string: string): string {
+  const midPoint = Math.floor(string.length / 2);
+  let mixedString = "";
+
+  for (let i = 0; i < string.length; i++) {
```

Comment:
if you're going to reverse both halves, couldn't you just reverse the whole thing and then return the two slices?

```
  const midPoint = Math.floor(string.length / 2);
  const reversed = string.split("").reverse().join("");
  return reversed.substring(midPoint) + reversed.slice(0, midPoint);
```

Reply:
Great catch! Will fine-tune. ✨🔧✨